manual_test_guides.md

```diff
@@ -0,0 +1,52 @@
+O3DE Manual Test Guides
+--
+The following test guides are recommended to be used for manual test verification of any changes made to their corresponding features, components and systems. Due to the complexity of the O3DE product and tools it has been determined that automated tests alone are not sufficient to fully mitigate the risk of escaped defects in submitted changes. The purpose of these manual test workflows are to serve as a supplement to the Automated Review test system and have been proven effective at uncovering defects not found through automated tests.
+
+Each workflow test suite is owned and maintained by the SIG which owns their corresponding feature, component and system, the Testing SIG is therefore only responsible for maintaining this compiled list of test suites as reference documentation. Workflow suites tagged with Last Known Good (LKG) are recommended to be used as the bar for establishing a LKG tagged build, suites without the LKG tag can still be run but are not considered blocking for the LKG process.
```

Comment:
May be worth calling out SIG-Testing as available to consult with on the content or format, but not directly responsible for other sig's areas. (essentially the [involved vs. committed parable](https://www.visual-paradigm.com/scrum/scrum-pig-and-chicken/) from the agile mythos)

This doesn't do much to inform the how or why of marking certain workflows as LKG. Is more-specific LKG guidance documented somewhere else that can be linked to? Should it be documented here?